config/services/gpsd-server.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>gpsd-server</short>
```

Comment:
AFAICS, `gpsd` always behaves as a server. As such we can call this service `gpsd` instead of `gpsd-server`.

Reply:
yes, thats true. i changed it.